src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -401,8 +404,11 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals, too-many-statements
 
     step = task.step
     branch = None
+
     if context_path is None:
         context_path = step.context_path
```

Comment:
Is it possible the  step.context_path = '/dev/null'? Should we set context_path to None as well?

Reply:
Is it possible the step.context_path = '/dev/null'? 
No, in create: `if context_path.lower() == ACR_NULL_CONTEXT:
        context_path = None` so if it were set to /dev/null during creation, it will be converted to None.

Should we set context_path to None as well?
Yes, given context_path is checked following the same way as create.